appcues/src/main/java/com/appcues/ui/presentation/EmbedViewPresenter.kt

```diff
@@ -15,7 +16,7 @@ internal class EmbedViewPresenter(
     private val stateMachines: StateMachineDirectory,
 ) : ViewPresenter(scope, renderContext) {
 
-    override fun ViewGroup.setupView(): ComposeView? {
+    override fun ViewGroup.setupView(activity: Activity): ComposeView? {
         return stateMachines.getFrame(renderContext)?.setupComposeView()
```

Comment:
should the `activity` pass through to `setupComposeView()` here? I guess it does not matter, as the `AppcuesFrameView` is already created with a `context` that was used to create the internal `composeView` there.

Reply:
no need, its using whatever it is coming from the original layout